examples/cmd/logtransport.go

```diff
@@ -1,17 +1,16 @@
 package main
 
 import (
-	"bytes"
-	"io/ioutil"
 	"net/http"
 	"time"
 
 	"gopkg.in/src-d/go-log.v1"
+
+	"github.com/src-d/metadata-retrieval/utils"
```

Comment:
In other reviews, we put the metadata-retrieval imports in the middle. I am guessing we have to come up with a linter. CC @lwsanty

Reply:
cc93eaf01d21c024cc960f30b68a9fa61dfeb933 thanks